apps/settings/test/marionette/tests/screen_lock_settings_test.js

```diff
@@ -98,8 +97,7 @@ marionette('manipulate screenLock settings', function() {
       'passcode is not checked');
   });
 
-  test(
-    'passcode is enabled and won\'t get disabled if you tap back button ' +
+  test('passcode is enabled and won\'t get disabled if you tap back button ' +
     'when we try to disable passcode directly',
     function() {
```

Comment:
This test is giving me intermittent error on my local machine. Not sure if you are experiencing it too? It seems like a slight time delay before the second togglePasscodeLock() on line 109 would fix the problem.